Cargo.toml

```
@@ -82,7 +82,7 @@ datafusion-physical-expr = { git = "https://github.com/apache/arrow-datafusion.g
 datafusion-sql = { git = "https://github.com/apache/arrow-datafusion.git", rev = "26e43acac3a96cec8dd4c8365f22dfb1a84306e9" }
 datafusion-substrait = { git = "https://github.com/apache/arrow-datafusion.git", rev = "26e43acac3a96cec8dd4c8365f22dfb1a84306e9" }
 derive_builder = "0.12"
-etcd-client = "0.12"
+etcd-client = "0.12.2"
```

The upstream bumps patch version only for new features. We'd better specify the patch version to avoid unexpected upragde.

---

Actually in cargo `0.12.2` means any version `<0.13, >= 0.12.2`. If we really want fixed version of etcd-client, we will need to pin `=0.12.2` but that's not best practice.

---

OK. So let's revert to `0.12`?

---

```suggestion
etcd-client = "0.12"
```

If so, please help apply this suggestion - I'll be back tomorrow morning :D